### DISCUSSION TITLE
# Fatal error in , line 0 - gatsby develop

### DISCUSSION THREAD

**Discussion by Musharofchy** [gatsbyjs/gatsby#29323] [Help]
Suddenly, **gatsby develop** returning an error. I am using M1 macbook pro with big sur, is there anyone faced same issue? if yes how to resolve this?. I have configured everything properly by following official doc, am I missing something?
<img width="563" alt="Screenshot 2021-02-04 at 4 19 05 AM" src="https://user-images.githubusercontent.com/1244883/106817342-efdd7280-66a0-11eb-80ac-51324ea6ee43.png">

Thanks

**Comment by reecebguida** [comment]
Hey @Musharofchy  I'm having the [same issue](https://github.com/netlify/cli/issues/1800)! No publicly available solution yet, it seems.

**Reply by Musharofchy** [reply]
Hey, Yeah. So disappointing, I thought gatsby have strong community but ... However, I fixed it by downgrading node. Please follow: https://stackoverflow.com/questions/66042288/nodejs-with-macos-fatal-error-in-line-0-gatsby-develop

**Reply by reecebguida** [reply]
Thank you for the creative solution! The recent release of Node.js v15.9.0 also fixes the issue. See here for more details: https://github.com/netlify/cli/issues/1800

**Reply by doublejosh** [reply]
I fixed this by using v.16 via nvm :)
